scripts/setup-systemd

```diff
@@ -16,4 +16,25 @@ Restart=always
 [Install]
 WantedBy=multi-user.target" > /etc/systemd/system/grist.service
 
+echo "[Unit]
+Description=Upgrade Docker Compose services for Grist
+After=network.target docker.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+WorkingDirectory=/home/grist
+ExecStart=/home/grist/bin/upgrade-grist"  > /etc/systemd/system/grist-upgrade.service
+
+echo "[Unit]
+Description=Run Docker Compose upgrade for Grist weekly
+
+[Timer]
+OnCalendar=Sun *-*-* 00:00:00
```

Comment:
Personally I like 04:00:00 for scheduled restarts, as that's the time fewest people seem to be around, but it also heavily depends on server timezone I guess!

Reply:
Sure, that seems like a better time for Europe and America. I've made that change.